docs/installation.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Installation Guide
+nav_order: 2
+---
+
+In order to use the MarkLogic Connector in an Anypoint project, you must add the connector to the project.
+This can be accomplished using the built-in assistant, which will find the connector in Anypoint Exchange and then add
+it to your project. Once the MarkLogic Connector has been successfully added to the Anypoint project, "MarkLogic"
+will appear as a module in the Mule Palette.
+
+## Installing from the Anypoint Exchange
```

Comment:
Remove "the" from "from the", and no need for a heading here since this page is now nice and short.